### PR TITLE
Add org.kosmospredanie.Yakblt

### PR DESCRIPTION
**Pull request by kosmospredanie**
Yakblt is a simple bullet-dodging game.

https://gitlab.com/kosmospredanie/yakblt

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build org.kosmospredanie.Yakblt

File: org.kosmospredanie.Yakblt.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "org.kosmospredanie.Yakblt",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--device=dri",
+		"--socket=pulseaudio",
+		"--filesystem=~/.yakblt-scores.txt"
```
Comment (bilelmoussaoui) [inline org.kosmospredanie.Yakblt.json]: Any reason for needing access to this?

File: org.kosmospredanie.Yakblt.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		"/lib/cmake",
+		"/lib/debug",
+		"/share/gir-1.0",
+		"/lib/girepository-1.0"
+	],
+	"modules": [
+		"shared-modules/gudev/gudev.json",
+		{
+			"name": "foo"
+		},
```
Comment (bilelmoussaoui) [inline org.kosmospredanie.Yakblt.json]: Can be removed.

File: org.kosmospredanie.Yakblt.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		{
+			"name": "sfml",
+			"buildsystem": "cmake",
```
Comment (bilelmoussaoui) [inline org.kosmospredanie.Yakblt.json]: Have you tried building any of those two cmake modules using cmake-ninja?

File: org.kosmospredanie.Yakblt.json (hunk 1 above)
Comment (kosmospredanie) [reply org.kosmospredanie.Yakblt.json]: The program saves game scores to the file.

Comment (bilelmoussaoui) [reply org.kosmospredanie.Yakblt.json]: You should $XDG_DIRS for that. https://docs.flatpak.org/en/latest/conventions.html#xdg-base-directories

**Comment by kosmospredanie** [comment]
bot, build org.kosmospredanie.Yakblt

**Pull request closed without merging**